### DISCUSSION TITLE
[3.5"] Razer Chroma for Turzx

### DISCUSSION THREAD

**Discussion by mosesman831** [mathoudebine/turing-smart-screen-python#363] [Themes]
This is the first theme I have ever created, so please give me some details on how I can improve this.

Details: For Turzx 3.5in
Version: 1.1

Preview:

![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/56060463/44f38468-1011-4b75-b407-ae0858e7415a)
![image](https://github.com/mathoudebine/turing-smart-screen-python/assets/56060463/995e2cb0-6f04-418c-9911-74ce0d919ff9)

[RazerChromaS1.1.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/13040891/RazerChromaS1.1.zip)

**Comment by AnthonyDimuzio69** [comment]
what do i  have to do besides download to use this theme?

**Reply by mosesman831** [reply]
You will have to open the TURZX APP and import the theme. Then enter some custom field texts to improve the looks.

Please keep in mind that this is the old version of Razer Chroma and have not been updated in a while.

**Comment by AnthonyDimuzio69** [comment]
I have downloaded it but doesn’t show up in files to use it Sent from my iPhoneOn Jul 29, 2024, at 9:22 AM, Moses Man ***@***.***> wrote:﻿
You will have to open the TURZX APP and import the theme. Then enter some custom field texts to improve the looks.
Please keep in mind that this is the old version of Razer Chroma and have not been updated in a while.

**Reply by mosesman831** [reply]
Could you try downloading again, with another browser maybe?

**Comment by AnthonyDimuzio69** [comment]
That wouldn’t change anything same download. When I open the folder of the download it’s not there Sent from my iPhoneOn Jul 29, 2024, at 9:46 AM, Moses Man ***@***.***> wrote:﻿
Could you try downloading again, with another browser maybe?

**Comment by AnthonyDimuzio69** [comment]
I just realized I don’t use that app because it never says it’s connected. I use another app that works for me for it🤷🏻‍♂️Sent from my iPhoneOn Jul 29, 2024, at 9:52 AM, Anthony Dimuzio ***@***.***> wrote:﻿That wouldn’t change anything same download. When I open the folder of the download it’s not there Sent from my iPhoneOn Jul 29, 2024, at 9:46 AM, Moses Man ***@***.***> wrote:﻿
Could you try downloading again, with another browser maybe?

**Reply by mosesman831** [reply]
I don't think using a 3rd party app would affect if the app downloads or not.
